lib/ReactViews/Tools/PedestrianMode/Marker.tsx

```diff
@@ -0,0 +1,40 @@
+import Cartesian3 from "terriajs-cesium/Source/Core/Cartesian3";
```

Comment:
Should this be a .tsx file? Since it's a Model, it probably doesn't belong in ReactViews, either.

Reply:
Good catch. I have renamed it to `.ts`, however the class is not generally useful so I don't want to take it out of context and place it in `Models/`.